mdsa.py

```diff
@@ -0,0 +1,116 @@
+import os
+import unittest
+from pprint import pprint, pformat
+
+import click
+import numpy as np
+
+from mdsa.algorithm import Algorithm
+from mdsa.algorithms.nystrom import Nystrom
+from mdsa.algorithms.scmds import Scmds
+
+# Register algorithms here
+Algorithm.register(Nystrom())
+Algorithm.register(Scmds())
+
+@click.group()
+def main():
+    click.echo('Multidimensional scaling approximations. For command line options, re-run with --help flag.')
+
+
+@click.command()
```

Comment:
should just be able to do `@main.command()` here instead of explicitly adding on line 35